plugins/module_utils/schema.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from collections import namedtuple
+
+
+class MSOSchema:
```

Comment:
any reason that we create a new class for MSOSchema and could it be reused in other modules as well?